File: src/test/samples/validSamples/validSample.lua
--#region FirstRegion
local x = 42
--#endregion

-- #region Second Region  
MyClass = {}
function MyClass:method()
    --    #region   InnerRegion  
  print("Hello")
    --         #endregion    ends InnerRegion  

  --  #region  
    print("Unnamed region")
--#endregion  
end
-- #endregion

File: src/test/samples/validSamples/validSample.jl
#region FirstRegion
x = 42
#endregion

# region Second Region  
struct MyClass
  #  region    InnerRegion   
  function method()
    println("Hello")
  end
  #  endregion    ends InnerRegion

    #region
    function method2()
      println("Unnamed region")
    end
    #   endregion
end
# endregion  
